src/mlpack/methods/neighbor_search/sort_policies/nearest_neighbor_sort.hpp

```diff
@@ -150,6 +150,21 @@ class NearestNeighborSort
       return DBL_MAX;
     return a + b;
   }
+
+  /**
+   * Return the given value relaxed.
+   *
+   * @param value Value to relax.
+   * @param epsilon Relative error (non-negative).
+   *
+   * @return double Value relaxed.
+   */
+  static inline double Relax(const double value, const double epsilon)
+  {
+    if (value == DBL_MAX)
+      return DBL_MAX;
+    return (1 / (1 + epsilon)) * value;
```

Comment:
In the furthest_neighborhood_sort.hpp? But (1 + epsilon) will be a linear function. I think the sensitivity of epsilon will differ from nearest neighbor to furthest neighbor.